src/coreclr/tools/Common/Compiler/ObjectWriter/ElfObjectWriter.cs

```diff
@@ -155,9 +158,10 @@ private protected override void CreateSection(ObjectNodeSection section, Utf8Str
             });
 
             // Emit section symbol into symbol table (for COMDAT the defining symbol is section symbol)
+            Utf8String sectionNameUtf8 = new(sectionName);
```

Comment:
I don't know if it wouldn't be better to leave the object writer and debug info for later. Many of these would be avoidable by more switching to Utf8String, but this is the kind of internal interface change that is better as a separate PR. Also, WASM object writer is in progress so this would be a relatively bad timing to do an internal interface change.

We could also say that all uses of the Utf8String constructor are unreviewed here if you really want the operator gone.